bin/olly.ml

```diff
@@ -345,7 +345,7 @@ let () =
         `Blocks help_secs;
       ]
     in
-    let doc = "Report the GC latency profile and stats." in
+    let doc = "Report the GC latency profile and stats. Total time spent by the program performing garbage collection (major and minor)" in
```

Comment:
Thanks for the PR @ju-sh! You can possibly add the metrics and descriptions mentioned here: https://github.com/tarides/runtime_events_tools#gc-stats